### PR TITLE
Taxonomy edits 5 - ingredient dedupes

### PR DESCRIPTION
**Pull request by svensven**
mostly from working through ` grep -oP '^en:[^,\r]+' ingredients.txt | sort | uniq -di` etc
(splitting the branch into additions and dedupes for clarity)

### CODE REVIEW

File: t/ingredients_processing.t [hunk 1]
```diff
@@ -833,7 +833,7 @@ my @tests = (
 				    'text' => 'Guaven'
 				  },
 				  {
-				    'id' => 'en:rosehip',
+				    'id' => 'en:rose hip',
```
Comment (Ban3) [inline t/ingredients_processing.t:836]: Spaces get replaced with `-`, so the id is `rose-hip`

Comment (svensven) [reply t/ingredients_processing.t]: i noticed *after* i'd already pressed commit :D

File: taxonomies/ingredients.txt [hunk 2]
```diff
@@ -5421,7 +5421,6 @@ pt:fermentos lácticos e de maturação
 en:selected lactic ferments
 de:ausgewählte Milchsäurekulturen
 fr:ferments lactiques sélectionnés
-nl:geselecteerde fermenten
```
Comment (aleene) [inline taxonomies/ingredients.txt:5424]: Looks ok to me, otherwise the other entry should be removed.

File: taxonomies/ingredients.txt [hunk 3]
```diff
@@ -66193,6 +65719,7 @@ fr:miel non raffiné, miel brut
 en:pure honey
 de:Purer Honig
 fr:Miel pur
```
Comment (aleene) [inline taxonomies/ingredients.txt:65721]: Does it have a parent? I can not see it on the web-interface.

**Review by aleene — APPROVED** [review]
Difficult to review, as I can not see the doubles.

File: taxonomies/ingredients.txt (hunk 2 above)
Comment (svensven) [reply taxonomies/ingredients.txt:5424]: the other instance is under
```<en:ferments
en:selected ferments
de:ausgewählte Kulturen
es:fermentos seleccionados
fr:ferments sélectionnés
it:fermenti selezionati
nl:geselecteerde fermenten
pt:fermentos selecionados
# ingredient/fr:ferments-sélectionnés has 134 products in 7 languages @2019-07-08
```
which seems like the more accurate translation.

File: taxonomies/ingredients.txt (hunk 3 above)
Comment (svensven) [reply taxonomies/ingredients.txt:65721]: The original entries were as follows:
```<en:honey
en:pure honey
de:Purer Honig
fr:Miel pur
```
and
```
<en:honey
fr:miel pur
# ingredient/fr:miel-pur has 3 products @2019-0-28
```

File: taxonomies/ingredients.txt (hunk 2 above)
Comment (aleene) [reply taxonomies/ingredients.txt:5424]: Yes you are right. You could change it to `geselecteerde melkzuurfermenten`.

**Comment by stephanegigandet** [comment]
Thank you!

**Pull request merged**